Makefile

```diff
@@ -22,7 +22,7 @@ export GOBIN
 
 .PHONY: dev
 dev:
-	@go run cmd/node-controller-manager/controller_manager.go --kubeconfig=dev/kubeconfig.yaml --v=2
+	@go run cmd/node-controller-manager/controller_manager.go --kubeconfig=dev/kubeconfig.yaml --kubeconfig-seed=dev/kubeconfig-seed.yaml --namespace=default --v=2
```

Comment:
Would it be better if we gave more understandable names for the kubeconfigs. Maybe _control-kubeconfig and target-kubeconfig_ ?

Reply:
I agree with Prashanth!

Reply:
Yup I would agree with that. `control-kubeconfig` and `target-kubeconfig` sounds good,  except I wanted to be little convenient to the user, not aware of separate control-k8-cluster, `kubeconfig` then could have helped. ;)
Will make the change soon.